modules/ding_ddbasic/modules/ding_ddbasic_opening_hours/ding_ddbasic_opening_hours.pages.inc

```diff
@@ -1,30 +1,37 @@
-<?php
-/**
- * @file
- * Ding DDBasic opening hours pages.
- */
-
-/**
- * Callback for fetching a week of opening hours.
- */
-function ding_ddbasic_opening_hours_week_callback($node, $from, $to, $categories) {
-  $commands = array();
-
-  if (opening_hours_present_on_node($node->nid)) {
-    if (!empty($categories)) {
-      $categories = explode('|', $categories);
-    }
-
-    $tmp = theme('ding_ddbasic_opening_hours_week', array(
-      'node' => $node,
-      'timespan' => array($from, $to),
-      'categories' => $categories,
-    ));
-  }
-
-  // Note that we trim the output, so Drupal doesn't add the surrounding div
-  // elements.
-  $commands[] = ajax_command_replace('.opening-hours-week-' . $node->nid, trim($tmp));
-
-  return array('#type' => 'ajax', '#commands' => $commands);
-}
+<?php
```

Comment:
Something is wrong here, since whole file is diff.

Reply:
The whole file is changed because of line ending which originally for this file was set to `CRLF` instead of traditional `LF`.

Reply:
I understand and appreciate the fix, however it's not related to PHP7.0 vs PHP7.2+.

Reply:
Fixed.